.buildkite/check-wheel-size.py

```diff
@@ -1,7 +1,7 @@
 import os
 import zipfile
 
-MAX_SIZE_MB = 100
+MAX_SIZE_MB = 400
```

Comment:
Let's don't go sky high right now and still be convervative a bit... (there is still project total limit)
```suggestion
MAX_SIZE_MB = 150
```